### ISSUE TITLE
Broken wheel build on windows (Nuget related?)

### ISSUE THREAD

**Issue by musicEnfanthen** [joerick/cibuildwheel#654]
Hi there, 

since two days we see an error when trying to build wheels via cibuildwheel within GH actions' Windows environment (macox and linux works fine).

Seems to happen during python installation with Nuget. The error can be found here: https://github.com/rism-digital/verovio/runs/2476403233?check_suite_focus=true#step:6:2989

It throws:

```
cibuildwheel version 1.9.0

Build options:
  platform: 'windows'
  architectures: {<Architecture.x86: 'x86'>, <Architecture.AMD64: 'AMD64'>}
  before_all: 'choco install swig -f -y'
  before_build: "swig -version && bash -c 'cd tools; ./get_git_commit.sh' && swig -c++ -python -py3 ./bindings/python/verovio.i"
  before_test: None
  build_selector: BuildSelector('cp36-win32' - 'cp27-*')
  build_verbosity: 0
  dependency_constraints: DependencyConstraintsWindowsPath('C:/Program Files (x86)/pipx/.cache/035e9047b631176/Lib/site-packages/cibuildwheel/resources/constraints.txt'))
  environment: ParsedEnvironment([])
  manylinux_images: None
  output_dir: WindowsPath('wheelhouse')
  package_dir: WindowsPath('.')
  repair_command: ''
  test_command: None
  test_extras: ''
  test_requires: []
  test_selector: TestSelector('*')

Here we go!

Running before_all...
                                                            ✓ 137.11s

Building cp36-win32 wheel
CPython 3.6 Windows 32bit

Downloading nuget...
  
  + Download https://dist.nuget.org/win-x86-commandline/latest/nuget.exe to C:\cibw\nuget.exe
                                                              ✓ 0.12s
Installing Python cp36...
  
  + C:\cibw\nuget.exe install pythonx86 -Version 3.6.8 -OutputDirectory C:\cibw\python
  Feeds used:
    C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
  
  
  
  Attempting to gather dependency information for package 'pythonx86.3.6.8' with respect to project 'C:\cibw\python', targeting 'Any,Version=v0.0'
  Package 'pythonx86 3.6.8' is not found in the following primary source(s): 'C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\'. Please verify all your online package sources are available (OR) package id, version are specified correctly.
Error: Command ['C:\\cibw\\nuget.exe', 'install', 'pythonx86', '-Version', '3.6.8', '-OutputDirectory', 'C:\\cibw\\python'] failed with code 1. None
                                                              ✕ 2.62s
```

There seems to be a related issue in the actions/environment repo: https://github.com/actions/virtual-environments/issues/3240

Is this a known issue? (The recommended workaround seems to be to downgrade nuget to 5.8.x, is this possible from outside cibuildwheel?)

**Comment by musicEnfanthen** [comment]
Thanks a lot for the super fast reply. We'll try this.

**Comment by lpugin** [comment]
Looks like it works https://github.com/rism-digital/verovio/runs/2478309065?check_suite_focus=true Brilliant, thanks!

**Comment by musicEnfanthen** [comment]
@joerick Yes, the fix from your PR did the trick. Thanks from me, too!

**Comment by henryiii** [comment]
@joerick we should probably get a release out in the next few days with this?

**Comment by joerick** [comment]
yes, agreed.

**Comment by joerick** [comment]
I just released v1.11.0 with this fix included.

**Comment by musicEnfanthen** [comment]
Thanks a lot!
